components/molecules/GamePlayedStory/GamePlayedStory.tsx

```diff
@@ -1,61 +1,91 @@
 import React from 'react'
-import {Pressable, StyleSheet, Text, View, ViewStyle} from "react-native"
+import {Pressable, ScrollView, StyleSheet, View} from "react-native"
 import GameCard from '../GameCard/GameCard'
-import {FriendCircle} from '../Friend/FriendGroup'
-import {FriendNetworkStatus} from '../Friend/Friend'
+import FriendGroup, {FriendCircle, FriendGroupItem} from '../Friend/FriendGroup'
+import Title from '../../atoms/Title/Title'
 
 export interface GamePlayedStoryProps {
-    title: string
-    style?: ViewStyle
-    gameBookImageUrl: string | undefined
-    gameBookTitle: string
-    onBookTapped: () => void
-    onStoryTapped: () => void
-    avatarImage: string | undefined
-    avatarUername: string
-    avatarStatus: FriendNetworkStatus
+    friends: FriendGroupItem[]
+    games: {imageUrl: string; name:string, onThumbnailTapped?: ()=> void}[]
+    timeString: string 
+    maxWidth?: number
```

Comment:
I don't think this is the right set of props to instantiate the component, but I'm not sure what it should be. I'm going to start a discussion thread in slack about this.

Reply:
@jeansampen Let’s follow this new idea you did with the `games` prop. Basically, let’s avoid Interfaces and instead define all the specific keys we need, just like you did. But here’s what I propose as the names:

```
friends: { firstName: string, imageUrl?: string, status: ‘online’ | ‘offline’, onTapped?: ()=>void }[]
games: { name: string, imageUrl: string; onTapped?: ()=> void}[]
timeAgo: string
maxWidth?: number
```

A couple implication of this: the friend avatars should be tappable. Also, I think we should just rename “networkStatus” to “status” so let’s make that change throughout the repo since we pass this around in other places.

Reply:
@jeansampen I realized one more wrinkle: In the spec above I wrote "onTapped" as an attribute of the friends array. However, in the event there are multiple friends, there should only be one onTapped.

Let me try restating that. On a rendered GamePlayedStory the friends avatar stack is a single tap target. Meaning, multiple friends will open that whole group when you tap it and a single friend will just open that single friend when you tap it.

But games are different: each game is rendered with it's own tap target. so the props should probably be:

```
friends: { firstName: string, imageUrl?: string, status: ‘online’ | ‘offline’ }[]
onFriendsTapped?: ()=>void
games: { name: string, imageUrl: string; onTapped?: ()=> void}[]
timeAgo: string
maxWidth?: number
```